src/Dev/Tasks/TagsToShortcodeTask.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace SilverStripe\Assets\Dev\Tasks;
+
+use SilverStripe\Dev\BuildTask;
+
+/**
+ * Class TagsToShortcodeTask
+ * @package SilverStripe\Assets\Dev\Tasks
+ */
+class TagsToShortcodeTask extends BuildTask
+{
+    private static $segment = 'TagsToShortcode';
+
+    protected $title = 'Rewrite tags to shortcodes';
+
+    protected $description = "
```

Comment:
I think this only gets shown on the web interface, so all your content will be rendered on a single line.